src/containers/genePage/alleleTable.js

```diff
@@ -313,8 +314,9 @@ const AlleleTable = ({ isLoadingGene, gene, geneId}) => {
           data={data}
           downloadUrl={`/api/gene/${geneId}/alleles/download`}
           keyField='id'
-          rowStyle={{cursor: 'pointer'}}
-          selectRow={selectRow}
+          //uncomment when sequence viewer is working again
```

Comment:
I would put the JIRA ticket in this comment as we know we need to turn this back on in the future.